addons/sale/tests/test_sale_to_invoice.py

```diff
@@ -533,6 +533,105 @@ def test_invoice_with_sections(self):
 
         self.assertEqual(invoice.line_ids[0].display_type, 'line_section')
 
+    def test_invoice_combo_product(self):
+        """ Test creating an invoice for a SO with a combo product. """
+        product_a = self._create_product(name="Product A")
+        product_b = self._create_product(name="Product B")
+        combo_a = self.env['product.combo'].create({
```

Comment:
I do not know who decided to make product.combo and product.combo.item anyways, but why can not you just make one combo instead of two?

Reply:
It actually makes more sense if A is frice and B is burger.

Reply:
A combo represents a set of selection options of which you can choose one (the selection options are the combo lines). 

In the burger example, you could have a 'Burger' combo for which the combo lines could be 'Normal Burger' or 'Horse-meat burger', and a 'Side' combo for which the combo lines could be 'French fries' or 'Salad'. 

Then, if you create a 'Meal menu' product that references both combos, that means the customer can choose one burger and one side.

In this case, I could have made just one combo and used it twice in the meal menu, but I think it made more sense to have a Burger combo and a Side combo like in the demo data.